torch/_decomp/decompositions.py

```diff
@@ -2226,6 +2226,12 @@ def numel_error():
 
     return (self * other).sum()
 
+@register_decomposition(aten.mul.left_t)
```

Comment:
That overload of the mul op does not exist in native_functions.yaml. What is it?

Reply:
@albanD I don't know why this exists but this is python semantics on `int*list` multiplication (i.e. self concat). it is registered op through prim ops

https://github.com/pytorch/pytorch/blob/56511f859a32aa4fc9648aa1ca69d0689e09bca3/torch/csrc/jit/runtime/register_prim_ops.cpp#L602

https://github.com/pytorch/pytorch/blob/56511f859a32aa4fc9648aa1ca69d0689e09bca3/torch/csrc/jit/runtime/register_ops_utils.cpp#L318

Reply:
This kinda looks like a Torchscript specific thing. I'm not sure when/why we'd use this decomp.